pkg/controller/discovery/web/web.go

```diff
@@ -0,0 +1,243 @@
+package web
+
+import (
+	"github.com/fusor/mig-controller/pkg/controller/discovery/container"
+	"github.com/fusor/mig-controller/pkg/controller/discovery/model"
+	"github.com/fusor/mig-controller/pkg/logging"
+	"github.com/fusor/mig-controller/pkg/settings"
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"regexp"
+	"strconv"
+	"time"
+)
+
+// Application settings.
+var Settings = &settings.Settings
+
+var Log *logging.Logger
+
+// Defaults
+const (
+	WaitDefault = time.Second * 10
+)
+
+const (
+	RootPath = "/namespaces/:namespace/clusters/:cluster/"
+)
+
+//
+// Web server
+type WebServer struct {
+	allowedOrigins []*regexp.Regexp
+	*container.Container
+}
+
+func (w *WebServer) Start() {
+	r := gin.Default()
+	r.Use(cors.New(cors.Config{
+		AllowMethods:     []string{"GET"},
+		AllowHeaders:     []string{"Origin"},
```

Comment:
For now, while the authorization is still in progress, to allow [UI client](https://github.com/fusor/mig-ui/pull/640) to work, this line should say
```golang
AllowHeaders:     []string{"Origin", "Authorization"},
```
to allow passing authentication token, otherwise server response returns error, as the `Authorization` field is not allowed in `OPTIONS` header, which identifies itself as a CORS issue.

Reply:
Good catch!  Fixed.